x-pack/filebeat/input/entityanalytics/provider/azuread/azure.go

```diff
@@ -212,10 +212,12 @@ func (p *azure) runIncrementalUpdate(inputCtx v2.Context, store *kvstore.Store,
 		return err
 	}
 
-	if updatedUsers.Len() != 0 || updatedDevices.Len() != 0 {
+	wantUsers := p.conf.wantUsers()
+	wantDevices := p.conf.wantDevices()
```

Comment:
@taylor-swanson I think we can omit these conditions here since the returned values form `doFetch` should be empty in the case that. Do you agree?